### PR TITLE
added troubleshooting to kool-share docs

### PR DESCRIPTION
**Pull request by bessa3301**
| Issue | # |
| -----: | :-----: |
| :beetle: Bug Fix | No |
| :trophy: Feature | No |
| :pencil: Refactor | No |
| :open_book: Documentation | Yes |
| :warning: Break Change | No |

**Description**

Included troubleshooting for `kool-share`

---

**Notes**

Please comment to help me enhance the text :)

### CODE REVIEW

**Comment by bessa3301** [comment]
maybe include slack link to join as well ?

**Review by fabriciojs — CHANGES_REQUESTED** [review]
@bessa3301 thank you very much! This is a great addition!

Just one issue though... the file you changed is generated automatically by `make-docs` command. So we need to figure out how to add this piece of documentation only to the generated kool-share.md file.

**Comment by danielsuguimoto** [comment]
@bessa3301 @fabriciojs I have an idea: why don't we create a troubleshooting section...or useful links section...then we link these items to our blog posts?

**Comment by fabriciojs** [comment]
I like @danielsuguimoto suggestion - we should start a new page in the documentation for troubleshooting, add this content there and then just add a link in the long description message pointing to that page in case of need.

**Comment by bessa3301** [comment]
This sounds awesome, I'll make these changes.

**Comment by danielsuguimoto** [comment]
@fabriciojs @dbpolito @bessa3301 can't we use a kooldev subdomain to redirect to the slack invite link...or any other way to make it static at least in this readme file?

**Comment by dbpolito** [comment]
good idea, as we have an alias for `kool.dev/install` we can make a `kool.dev/slack`

File: docs/4-Commands/kool-share.md [hunk 1]
```diff
@@ -23,5 +23,4 @@ kool share
 
 ### SEE ALSO
 
-* [kool](kool)	 - Cloud native environments made easy
-
+-   [kool](kool) - Cloud native environments made easy
```
Comment (danielsuguimoto) [inline docs/4-Commands/kool-share.md:26]: run `kool run make-docs` to regenerate it as it was before...

Comment (bessa3301) [reply docs/4-Commands/kool-share.md:26]: alright!

File: docs/6-Troubleshooting/intro.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+### Need any help?
+
+Kool is constantly evolving, but if you feel confused or lost on how to take full advantage of the tools Kool' has to offer, this section might be particularly useful. 
+
+Here you will find known fixes or workarounds for commom questions. Also, in the case that the information here contained doesn't help out you can join our community over [slack]()!
```
Comment (fabriciojs) [inline docs/6-Troubleshooting/intro.md:5]: we already have the `/slack` link alias, please get it with @dbpolito and update this to link to it properly.

**Review by fabriciojs — CHANGES_REQUESTED** [review]
I like how this turned out! Thanks @bessa3301 ! One minor detail on the Slack link so I can approve it.

File: docs/6-Troubleshooting/intro.md (hunk 2 above)
Comment (bessa3301) [reply docs/6-Troubleshooting/intro.md:5]: oooh thats cool. I didn't knew about the `/slack` link! Awesome!

Comment (bessa3301) [reply docs/6-Troubleshooting/intro.md]: > updated it as instructed.

**Review by bessa3301 — COMMENTED** [review]
weird... codecov failed by -0,01% o-o

**Review by danielsuguimoto — APPROVED** [review]

**Comment by bessa3301** [comment]
should I change something else ? :D

**Comment by danielsuguimoto** [comment]
@bessa3301 for me, its good to go...I'll merge it

**Pull request merged**